CONTRIBUTING.md

```diff
@@ -368,6 +369,27 @@ at the beginning of the line:
 I love it! Very clean code and great tests. lgtm.
 ```
 
+## Continuous Integration
+
+The Kata Containers project has a gaiting process to prevent introducing
```

Comment:
s/gaiting/gating/ :-) Although the CI does sometimes have a funny wobble to the way it walks, I don't think that is what we were documenting :-)